src/main/java/net/mcreator/ui/procedure/RetvalProcedureSelector.java

```diff
@@ -49,13 +49,14 @@
 import java.awt.*;
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
+import java.util.Objects;
 
 public abstract class RetvalProcedureSelector<E, T extends RetvalProcedure<E>> extends AbstractProcedureSelector {
 
 	@Nullable private final JComponent fixedValue;
 
 	public RetvalProcedureSelector(VariableType returnType, @Nullable IHelpContext helpContext, MCreator mcreator,
-			String eventName, Side side, @Nullable JComponent fixedValue, int width,
+			String eventName, Side side, boolean allowInlineEditor, @Nullable JComponent fixedValue, int width,
```

Comment:
This change is hard to read. No docs were provided and no explanation on what you did here.

Reply:
(or there were not many changes but idea is showing strange diff)

either way, some comments on PR and in Javadoc possibly would help :)

Reply:
You mean porting documentation I did on item properties pull request... Will do that.

Reply:
That would help, yep.

Diff is also strangely formatted (thanks github), did you only wrap the code with if for allowInlineEditor I presume?

Reply:
Yep.

Reply:
Then it is all good, no need for additional javadoc